### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'positionAbsolute')     at nodeToRect  while using getIntersectingNodes

### DISCUSSION THREAD

**Discussion by sidongdong** [xyflow/xyflow#3357] [Q&A]
I'm trying to make dynamic grouping possible.
So, first I tested getIntersectignNodes() to detect intersected nodes.
However 

TypeError: Cannot read properties of undefined (reading 'positionAbsolute')
    at nodeToRect 

is shown whenever I drag my custom nodes.
Why this error is shown and how can I solve it?

**Comment by moklick** [comment]
Hey! Could you create a codesandbox for this https://new.reactflow.dev/react-ts ?

**Comment by sarangsami** [comment]
i have used setTimeout and it worked
